cli/src/action/circuit/mod.rs

```diff
@@ -25,13 +25,15 @@ use std::fs::File;
 use std::io::Read;
 
 use clap::ArgMatches;
+use splinter::admin::messages::{CreateCircuit, SplinterService};
```

Comment:
The commit message should say without having to run`splinter circuit proposals` not list

Reply:
Fixed